### ISSUE TITLE
in game gang map system

### ISSUE THREAD

**Issue by minipunch** [minipunch/usarealismrp-fxserver#2209]
can use like colored circle blips to denote certain turfs

maybe with a command that toggles it on/off

**Comment by Crownapple** [comment]
https://store.rcore.cz/package/4913168
This is not bad and it can be toggled on and off

**Comment by minipunch** [comment]
> https://store.rcore.cz/package/4913168 This is not bad and it can be toggled on and off

Yeah, good looks. I was thinking about trying that one out actually.

Have you used it before or just seen the preview?

**Comment by Crownapple** [comment]
I have used it before. You take truf by selling drugs and putting tag in other gangs turfs

**Comment by Crownapple** [comment]
and you can collet money from business for protection
